app/controllers/downloaders_controller.rb

```diff
@@ -0,0 +1,20 @@
+class DownloadersController < ApplicationController
+  
+  def new
+    @downloader = Downloader.new  
+    @pictures = @downloader.pictures.build
+  end
+
+  def create
+    @downloader = Downloader.new(downloader_params)
+    if @downloader.save
+      @downloader.download_images(params[:pictures]['image'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.